src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -17,11 +17,13 @@ import { spawnSync } from "child_process";
 import * as path from "path";
 import * as admin from "firebase-admin";
 
-function slowRequireResolve(moduleName: string, opts?: { paths: string[] }): string {
-  opts = opts || { paths: [] };
+let app: admin.app.App;
+let adminModuleProxy: typeof admin;
+
+function slowRequireResolve(moduleName: string, cwd?: string): string {
```

Comment:
Can you just add a comment to this method explaining:
  * Why it's "slow"
  * When you should use it